addons/website/views/snippets/s_alert.xml

```diff
@@ -12,19 +12,24 @@
 
 <template id="s_alert_options" inherit_id="website.snippet_options">
     <xpath expr="." position="inside">
-        <div data-selector=".s_alert">
+        <div data-selector=".s_alert" data-js="Alert">
             <we-colorpicker string="Color"
                 data-select-style="true"
                 data-css-property="background-color"
                 data-color-prefix="alert-"/>
-        </div>
-
-        <div data-selector=".s_alert">
             <we-select string="Size">
                 <we-button data-select-class="s_alert_sm">Small</we-button>
                 <we-button data-select-class="s_alert_md">Medium</we-button>
                 <we-button data-select-class="s_alert_lg">Large</we-button>
             </we-select>
+            <we-select string="Type" data-apply-to=".fa.s_alert_icon">
+                <we-button data-select-class="fa-user-circle" data-select-type="alert-primary">Primary</we-button>
+                <we-button data-select-class="fa-user-circle-o" data-select-type="alert-secondary">Secondary</we-button>
+                <we-button data-select-class="fa-info-circle" data-select-type="alert-info">Info</we-button>
+                <we-button data-select-class="fa-check-circle" data-select-type="alert-success">Success</we-button>
+                <we-button data-select-class="fa-exclamation-triangle" data-select-type="alert-warning">Warning</we-button>
+                <we-button data-select-class="fa-exclamation-circle" data-select-type="alert-danger">Danger</we-button>
+            </we-select>
```

Comment:
We discussed with SBU, reorganize the options to have type, width, size, color in that order.

Reply:
If you want that exact order I'll have to split the options in 2 xpath to have some before and some after generic options Width (not specific to s_alert) :nauseated_face: 
Let's go I guess

Reply:
Mmh... too bad but yes "type" should really be the one the users use first and color as a last resort.